src/Clock/Clock.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+export class Clock extends React.Component {
+  state = {
+    date: '',
+  };
+
+  timerId?: NodeJS.Timer;
+
+  componentDidMount() {
+    this.timerId = setInterval(() => {
+      const date: Date = new Date();
+
+      this.setState({ date: date.toLocaleTimeString() });
+    }, 1000);
```

Comment:
Change the Clock component to print the time not only on the page but also in the DevTools console.